.github/workflows/check_pr_plugin_aws.yml

```diff
@@ -30,19 +30,20 @@ jobs:
         uses: actions/cache@v2
         with:
           path: ~/.cache/pip
```

Comment:
According to https://python-poetry.org/docs/configuration/#cache-dir poetry uses this path: `~/.cache/pypoetry`.
Maybe we should define `~/.cache` here.
Same is true for all other python GitHub actions.